client/my-sites/sharing/connections/connection.jsx

```diff
@@ -203,7 +228,7 @@ export default connect(
 		return {
 			siteId,
 			userHasCaps: canCurrentUser( state, siteId, 'edit_others_posts' ),
-			userId: getCurrentUserId( state ),
+			userId: getCurrentUserId( state )
```

Comment:
although we don't have this in our style guide, I highly encourage adding the trailing comma and I think a few others do as well. it certainly makes for less confusing diffs